src/pathfora.js

```diff
@@ -1232,27 +1236,33 @@
 
         // Hide fields
         Object.keys(config.fields).forEach(function (field) {
-          var parent, prev, next,
-              element = getFormElement(field);
-
-          if (element && !config.fields[field]) {
-            parent = element.parentNode;
-            prev = element.previousElementSibling;
-            next = element.nextElementSibling;
-
-            if (parent) {
-              // NOTE: collapse half-width inputs
-              if (element.className.indexOf('pf-field-half-width') !== -1) {
-                if (prev && prev.className.indexOf('pf-field-half-width') !== -1) {
-                  utils.removeClass(prev, 'pf-field-half-width');
-                }
+          var element = getFormElement(field),
+              parent = element.parentNode;
+
+          if (element && !config.fields[field] && parent) {
+            parent.removeChild(element);
+          }
+        });
+
+        // NOTE: collapse half-width inputs
+        Array.prototype.slice.call(widget.querySelectorAll('form .pf-field-half-width')).forEach(function (element, halfcount) {
```

Comment:
`forEach` doesn't work in IE < 9. Not sure what support we are shooting for, but this is easy enough to turn into a loop for compatibility.